connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -582,4 +626,161 @@ public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
             messageBatch.clear();
         }
     }
+
+    static class SinkTaskMetricsGroup {
+        private final MetricGroup metricGroup;
+        private final Sensor sinkRecordRead;
+        private final Sensor sinkRecordSend;
+        private final Sensor sinkRecordLag;
+        private final Sensor partitionCount;
+        private final Sensor offsetSeqNum;
+        private final Sensor offsetCompletion;
+        private final Sensor offsetCompletionSkip;
+        private final Sensor putBatchTime;
+        private final AtomicReference<Map<TopicPartition, OffsetAndMetadata>> consumedOffsets = new AtomicReference<>();
+        private final AtomicReference<Map<TopicPartition, OffsetAndMetadata>> committedOffsets = new AtomicReference<>();
+
+        public SinkTaskMetricsGroup(ConnectorTaskId id, ConnectMetrics connectMetrics) {
+            consumedOffsets.set(new HashMap<TopicPartition, OffsetAndMetadata>());
+            committedOffsets.set(new HashMap<TopicPartition, OffsetAndMetadata>());
+            metricGroup = connectMetrics.group("sink-tasks-metrics",
```

Comment:
same question as other pr -- this is `sink-task-metrics` instead of `sink-tasks-metrics` in the KIP

Reply:
yes. I'll correct.